### ISSUE TITLE
Late button- description on the attendance recoding page is not matching

### ISSUE THREAD

**Issue by poojaustad** [UstadMobile/UstadMobile#344]
**Describe the bug**
The late icon is mentioned as a partial tick-box on the attendance recording page instead of the late tick box 

**To Reproduce**
1. Talkback is enabled
2. Teacher user on attendance page to record new occurrence
3. When the cursor is on the Late button icon - it says a partial tick box

**Expected behavior**
On the attendance recording page, the late button should be called a late tick box 

Noted on andr
![2022-06-27_11h33_06](https://user-images.githubusercontent.com/78338755/175884885-22dc7592-e52a-4aef-b4c5-fa65538ec751.png)
oid device

**Comment by mikedawson** [comment]
This is intended behavior: partial attendance includes situations where the student arrives late or leaves early.
